pkg/config/setup/config.go

```diff
@@ -944,7 +944,9 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	bindEnvAndSetLogsConfigKeys(config, "sbom.")
 
 	// Synthetics configuration
-	config.BindEnvAndSetDefault("synthetics.enabled", false)
+	config.BindEnvAndSetDefault("synthetics.collector.enabled", false)
+	config.BindEnvAndSetDefault("synthetics.collector.workers", 5)
+	config.BindEnvAndSetDefault("synthetics.collector.flush_interval", "60s")
```

Comment:
Why `60s`, I think that's quite long interval, which can delay the time needed to report results to backend.

I would suggest something lower like `10s`, for reference, the Agent Integration scheduler uses `15s` and the Network Path Collector uses `10s` flush interval.